django/db/models/enums.py

```diff
@@ -8,7 +8,7 @@
 class ChoicesMeta(enum.EnumMeta):
     """A metaclass for creating a enum choices."""
 
-    def __new__(metacls, classname, bases, classdict):
+    def __new__(metacls, classname, bases, classdict, **kwds):
```

Comment:
I can't see why we're not matching the [new signature](https://github.com/python/cpython/commit/7aaeb2a3d682ecba125c33511e4b4796021d2f82#diff-e5dd6e444e4e6c2cdeb8edf271d057f9da52f7b406dcd8b655dc396fb013bd03R381) of the superclass here? 

```suggestion
    def __new__(metacls, classname, bases, classdict, boundary=None, **kwds):
```

(We should have had `kwds` before I guess…)

Reply:
Because older versions.

Reply:
Yeah, exactly.